src/domain/aspect/views/AspectDashboardView.tsx

```diff
@@ -171,7 +168,6 @@ const AspectDashboardView: FC<AspectDashboardViewProps> = props => {
                   placeholder={t('pages.aspect.dashboard.comment.placeholder')}
                   onPostComment={onPostComment}
                   maxLength={MID_TEXT_LENGTH}
-                  userAvatarUri={user?.user?.profile?.avatar?.uri}
```

Comment:
why was this required?

Reply:
Before the restyling the `PostComment` component was just a Markdown without the user avatar in it. After Aleks's changes it has the avatar on the left of the input field.
This is some code that I let Aleks merge to refactor it later because Neil told us to close Aleks'es task and let me continue on this other one. 
You can see his changes here: https://github.com/alkem-io/client-web/pull/2549/files
He added the userContext to the AspectDashboardView  only to pass the userAvatar to the PostComment form field. 
I didn't like that very much so I have moved that code to the PostComment.

After my last changes PostComment is renamed to PostMessageToCommentsForm